crates/hyperswitch_domain_models/src/router_request_types/authentication.rs

```diff
@@ -91,6 +91,8 @@ pub struct ConnectorAuthenticationRequestData {
     pub threeds_method_comp_ind: api_models::payments::ThreeDsCompletionIndicator,
     pub three_ds_requestor_url: String,
     pub webhook_url: String,
+
+    pub minor_amount: Option<MinorUnit>
```

Comment:
wouldn't `amount` be a better name for this field? 
The type itself will indicate that the amount is of type MinorUnit.

Reply:
> wouldn't `amount` be a better name for this field? The type itself will indicate that the amount is of type MinorUnit.
> 
> And there is already an `amount` field in this struct. Please use that field itself

Sorry for the confusion here. I wanted you to change type of the `amount` field here from `Option<i64>` to `Option<MinorUnit>` and then use it.